models/gpg_key_commit_verification.go

```diff
@@ -121,6 +122,11 @@ func ParseCommitWithSignature(c *git.Commit) *CommitVerification {
 		}
 	}
 
+	// If this a SSH signature handle it differently
+	if strings.Contains(c.Signature.Signature, "SSH") {
```

Comment:
What are the possible values of `c.Signature.Signature`? Would checking the substring `SSH` get incorrect result? eg: can `SSH` appear in a base64 string?

Reply:
Yes it could, I've added exact matching in last commit.